src/main/java/uk/gov/companieshouse/service/AppealService.java

```diff
@@ -83,44 +84,59 @@ private void createContactInChips(Appeal appeal, String userId) {
     }
 
     protected ChipsContact buildChipsContact(Appeal appeal) {
-
         final String companyNumber = appeal.getPenaltyIdentifier().getCompanyNumber();
         final OtherReason otherReason = appeal.getReason().getOther();
+        final IllnessReason illnessReason = appeal.getReason().getIllnessReason();
 
         final ChipsContact chipsContact = new ChipsContact();
         chipsContact.setCompanyNumber(companyNumber);
         chipsContact.setDateReceived(appeal.getCreatedAt().format(DATE_TIME_FORMATTER));
+        ReasonType reasonType = appeal.getReason().getReasonType();
 
-        final String contactDescription = "Appeal submitted" +
+        String contactDescription = "Appeal submitted" +
             "\n\nYour reference number is your company number " + companyNumber +
             "\n\nCompany Number: " + companyNumber +
             "\nEmail address: " + appeal.getCreatedBy().getEmailAddress() +
-            "\n\nAppeal Reason" +
-            "\nReason: " + otherReason.getTitle() +
-            "\nFurther information: " + otherReason.getDescription() +
-            "\nSupporting documents: " + getAttachmentsStr(appeal.getId(), otherReason);
-
+            "\n\nAppeal Reason";
+
+        List<Attachment> attachmentList;
+
+        switch(reasonType.getReasonType()){
```

Comment:
This would be cleaner and avoid the need for the base/abstract class, enum and switch statement if we used a utility class (single responsibility principle). Keep this service to just handling the logic and the utility class can handle all the description formatting. It should be easier to unit test too.

e.g. something like:

**Option 1: Simple utility class**
Service class:
```
String contactDescription = contactDescriptionFormatter.format(appeal);
```

Format method in utility class:
```
StringBuilder builder = new StringBuilder();
builder.append(formatCompanyDetails(companyNumber, emailAddress));

Reason reason = appeal.getReason();
if(reason != null) {
    if(reason.getOtherReason() != null) {
        builder.append(formatOtherReason(reason.getOtherReason()));
    }

    if(reason.getIllnessReason() != null) {
        builder.append(formatIllnessReason(reason.getIllnessReason()));
    }
}

return builder.toString();
```

Inside of the utility class you could have a formatAttachments() method too to avoid duplication.

**Option 2: Builder**
```
new ContactDescriptionBuilder(companyNumber, emailAddress)
    .setOtherReason(appeal.getReason().getOtherReason())
    .setIllnessReason(appeal.getReason().getIllnessReason())
    .build();
```

This option is dependent on the validator checking `appeal.getReason()` is not null otherwise it will throw a NullPointerException here and null checks within the build making this a less ideal solution but it is more readable.

Reply:
Builders came up in a discussion for improving the test data, however it was considered to large a change to make at the time.
If we do use option 2 we could simplify those bits of code as well.

Reply:
The base class, reason types and switch statement have been removed. Refactoring to extract the formatting code will be completed in another PR to avoid this PR getting more complex.